docs/hoon/twig/tis-flow/fas-var.md

```diff
@@ -11,7 +11,19 @@ noun with the subject.
 ## Expands to
 
 ```
-?@(p :pin(:name(p q) r) :pin(:cast(:coat(p q) r)
+:ifat  p
+  :pin  :name(p q)
+  r
+:pin  :cast(:coat(p) q)
+r
+```
+
+```
+?@  p
+  =+  p=q
+  r
+=+  ^-($=(p) q)
+r
 ```
```

Comment:
Note: the parenthetical nesting of the initial expansion is wrong. Also, this is a good example of the way these expansions can be confusing. `:coat(p)` isn't valid, but neither is `:coat(p q)`. I understand the actual expansions are macros on the AST ...